src/src/DataStructs/C013_p2p_dataStructs.cpp

```diff
@@ -42,7 +42,8 @@ bool C013_SensorDataStruct::matchesPluginID(pluginID_t pluginID) const
 
 bool C013_SensorDataStruct::matchesSensorType(Sensor_VType sensor_type) const
 {
-  if (sensorType == Sensor_VType::SENSOR_TYPE_NONE) {
+  if (sensorType == Sensor_VType::SENSOR_TYPE_NOT_SET || 
```

Comment:
Unfortunately that doesn´t fix it for me.
Still get a "SensorType mismatch" with that

Reply:
OK, thanks for testing, then I will change it to what you proposed.